packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -202,13 +202,8 @@ const styles: ComponentStyles<SiteManagedClasses, DevSiteDesignSystem> = {
         "&:-ms-expand": {
             display: "none",
         },
-        "&:hover": {
-            boxShadow: `inset 0 0 ${toPx(2)} 0 rgba(0,0,0, .3)`,
-        },
         "&:focus": {
-            boxShadow: (config: DevSiteDesignSystem): string => {
-                return `inset 0 0 0 1 ${config.brandColor}`;
-            },
+            boxShadow: "inset 0 0 0 1px #FB356D",
```

Comment:
Wondering why we have colors defined in the design system but we aren't using them?

Reply:
Site level design system values are not working. The design system is null. so any design system declarations on this page were not working.

Reply:
Keeping as is until the design system issue is fixed.